### DISCUSSION TITLE
Block Edition per record for specific users only

### DISCUSSION THREAD

**Discussion by shawkatmarrouf** [YetiForceCompany/YetiForceCRM#16786] [How to]
I want to block edition for specific users only.

Example: in project tasks, I would like to restrict SOME OF users who have the authority to edit tasks to be blocked from Editing task after condition like statue is complete

**Comment by Marshal4582** [comment]
Would be absolutely useful!

**Comment by rskrzypczak** [comment]
Check workflow and set whatever conditions you want.
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/1f331dc6-590a-40da-ae92-527282889771)
